package.json

```diff
@@ -88,14 +87,14 @@
     "eslint-plugin-react-hooks": "^2.3.0",
     "file-loader": "^6.2.0",
     "html-loader": "^0.5.5",
-    "html-webpack-plugin": "^5.6.0",
+    "html-webpack-plugin": "^4.5.2",
```

Comment:
Why was this downgraded instead of upgraded>

Reply:
New version of library is conflicting with Webpack v4 and works fine with old version

<img width="2056" alt="html-webpack-plugin" src="https://github.com/wri/gfw-mapbuilder/assets/855512/d593a639-469e-4385-92f7-339dd1b1198e">